ebpf/event_common.go

```diff
@@ -93,6 +93,22 @@ type ConnectionStats struct {
 	Type      ConnectionType      `json:"type"`
 	Family    ConnectionFamily    `json:"family"`
 	Direction ConnectionDirection `json:"direction"`
+	Conntrack *Conntrack          `json:"conntrack"`
+}
+
+// TODO: add to String()
+//easyjson:json
+type Conntrack struct {
```

Comment:
Might be a nit, but I don't like the idea of naming this Conntrack. Can we call it something like `IPTranslation`, `AddrTranslation`, or `Translation`?

Reply:
I went with IPTranslation.